deployment/stacks/data.ts

```diff
@@ -79,26 +79,29 @@ export function createDataComponents(props: CreateDataComponentsProps) {
         "service-role/AWSLambdaVPCAccessExecutionRole"
       ),
     ],
+    inlinePolicies: {
```

Comment:
This and in one other spot I added this minor refactor to add the inline policies directly to the resources.